nucypher/blockchain/eth/sol/source/contracts/lib/AdditionalMath.sol

```diff
@@ -93,4 +93,26 @@ library AdditionalMath {
         assert(b <= a);
         return a - b;
     }
+
+    /**
+    * @dev Adds signed value to unsigned value, throws on overflow.
```

Comment:
Thank you for including the exception in the docstring.  I think this is a fantastic docstring characteristic - especially for solidity.